CircleSpawn/CircleSpawn/CircleSpawnController.swift

```diff
@@ -1,11 +1,67 @@
 import UIKit
 
-class CircleSpawnController: UIViewController {
+class CircleSpawnController: UIViewController, UIGestureRecognizerDelegate {
+    
+    override func loadView() {
+        view = UIView()
+        view.backgroundColor = .white
+    }
+    
+    var locationInSubView = CGPoint()
+    let animation = Animation()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let doubleTap = UITapGestureRecognizer(target: self, action: #selector(handleDoubleTap(_:)))
+        doubleTap.numberOfTapsRequired = 2
+        doubleTap.delegate = self
+        view.addGestureRecognizer(doubleTap)
+    }
+    
+    func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRequireFailureOf otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        return true
+    }
```

Comment:
Bardzo niefajna implementacja tej metody. Wszystkie gesty wymagają, żeby wszystkie inne failowały, co powoduje że prosty pan przestanie działać na przykład.

Zamiast tego powinniśmy zobaczyć, że `gestureRecognizer` to nasz double tap, a `otherGestureRecognizer` to nasz triple tap.